Patterns/pattern6.py

```diff
@@ -1,17 +1,18 @@
 # Python code to print the following alphabet pattern
-#A 
-#B B 
-#C C C 
-#D D D D 
-#E E E E E   
+# A
+# B B
+# C C C
+# D D D D
+# E E E E E
 def alphabetpattern(n):

```

Comment:
The Snake formatting is commonly used in `Python` not an issue but one way to make it easier for Python Devs
`Snake format` utilizes underscores to link the words
```python
def alphabet_pattern():
```